Feature/more emojis

Mr1BitL8r opened the pull request:
Added more emojis for the English template and introduced the idea of using unique names that match the names used on https://emojipedia.org/halloween/ so that it is easier to identify, add and later also add more translations.

Mr1BitL8r committed:
Merge pull request #1 from LucasCtrl/main

Merging changes from upstream

Commit:
Added more emojis for English and introduced the idea of uniqueNames so the emojis and some infor about them can be found on the internet.

Commit:
Added bugfix for more emojis for English.

Commit:
Added bugfix for more emojis for English by removing the last ',' after the entry 'zombie'.

Commit:
Further bugfixes for more emojis for English by adding missing ',' and rearranging some entries.

tmttn commented:
Whoa! 😮

illusi0nary commented on lang/en.js:
```diff
@@ -1,42 +1,422 @@
+    {
+      name: 'heart',
+      uniqueName: 'Anatomical Heart',
+      emoji: '🫀',
```
This seems to be correct - is that only me ? 
I just see 🫀 - that might cause problems?

Mr1BitL8r replied:
Thanks for the hint. Some emojis are too new (e.g. "headstone" as well) so I changed them into comments and added the hint that they arrive later this year wit Unicode/Emoji 13.0, e.g. for Android 11.

illusi0nary commented on lang/en.js:
```diff
@@ -1,42 +1,422 @@
+    {
+      name: 'disguises',
+      uniqueName: 'Performing Arts',
+      emoji: '🥸',
```
Same here it is only showing 🫀

Mr1BitL8r replied:
The "Performing Arts" one actually was a copy-paste issue and should be fixed now. :)

Mr1BitL8r committed:
Merge branch 'main' into feature/more_emojis

Mr1BitL8r committed:
Update en.js

Bugfix as there was one '[' too much due to searching and replacing to get arrays for all "name" entries.

Mr1BitL8r committed:
Update en.js

And removed an additional '[' but now I should  have them... ^^'

Mr1BitL8r committed:
Update en.js

Most of the not displayed emojis are too new so I changed the ones now into a comment and replaced the "Performing Arts" one as it seemed to have been a copy-paste issue. :/

illusi0nary reviewed (approved):
All good :-)

LucasCtrl reviewed (approved):
I'm going to merge the changes but I'll delete the unique ID part and the comment one.
I the future, we are maybe going to change the file structure
